controllers/chat/group/removeGroupMember.js

```diff
@@ -57,7 +57,7 @@ const removeGroupMember = async (req, res) => {
     let channelResponse;
     try {
       const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
-        await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
+        await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `true` to `false` in the `updateBetterChannelMembers` function call could potentially affect the behavior of the function. If the function relies on this flag to perform certain operations for group channels, it might not work as expected after this change. Please ensure that the function can handle both cases correctly and that this change has been thoroughly tested.
```diff
-        await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
+        await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->